registry/coder-labs/modules/cursor-cli/main.tf

```diff
@@ -0,0 +1,263 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    coder = {
+      source  = "coder/coder"
+      version = ">= 2.7"
+    }
+  }
+}
+
+variable "agent_id" {
+  type        = string
+  description = "The ID of a Coder agent."
+}
+
+data "coder_workspace" "me" {}
+
+data "coder_workspace_owner" "me" {}
+
+variable "order" {
+  type        = number
+  description = "The order determines the position of app in the UI presentation. The lowest order is shown first and apps with equal order are sorted by name (ascending order)."
+  default     = null
+}
+
+variable "group" {
+  type        = string
+  description = "The name of a group that this app belongs to."
+  default     = null
+}
+
+variable "icon" {
+  type        = string
+  description = "The icon to use for the app."
+  default     = "/icon/cursor.svg"
+}
+
+variable "folder" {
+  type        = string
+  description = "The folder to run Cursor CLI in."
+}
+
+variable "install_cursor_cli" {
+  type        = bool
+  description = "Whether to install Cursor CLI."
+  default     = true
+}
+
+variable "enable_agentapi" {
+  type        = bool
+  description = "Whether to enable the AgentAPI for Cursor CLI."
+  default     = false
+}
+
+variable "install_agentapi" {
+  type        = bool
+  description = "Whether to install AgentAPI."
+  default     = true
+}
+
+variable "agentapi_version" {
+  type        = string
+  description = "The version of AgentAPI to install."
+  default     = "v0.4.0"
+}
+
+# Running mode is non-interactive by design for automation.
+
+
+variable "force" {
+  type        = bool
+  description = "Pass -f/--force to allow commands unless explicitly denied."
+  default     = false
+}
+
+variable "model" {
+  type        = string
+  description = "Pass -m/--model to select model (e.g., sonnet-4, gpt-5)."
+  default     = ""
+}
+
+variable "ai_prompt" {
+  type        = string
+  description = "AI prompt/task passed to cursor-agent."
+  default     = ""
+}
+
+variable "api_key" {
+  type        = string
+  description = "API key (sets CURSOR_API_KEY env or pass via -a)."
+  default     = ""
+  sensitive   = true
+}
+
+variable "mcp_json" {
+  type        = string
+  description = "Workspace-specific MCP JSON to write to ~/.cursor/mcp.json. See https://docs.cursor.com/en/context/mcp#using-mcp-json"
```

Comment:
Update description here and in script/tests to reflect we are soing it for project now instead of global.